webapp/channels/src/i18n/en.json

```diff
@@ -2179,6 +2179,8 @@
   "admin.service.iconTitle": "Enable integrations to override profile picture icons:",
   "admin.service.insecureTlsDesc": "When true, any outgoing HTTPS requests will accept unverified, self-signed certificates. For example, outgoing webhooks to a server with a self-signed TLS certificate, using any domain, will be allowed. Note that this makes these connections susceptible to man-in-the-middle attacks.",
   "admin.service.insecureTlsTitle": "Enable Insecure Outgoing Connections: ",
+  "admin.service.integrationRequestDesc": "The number of seconds to wait for Integration requests. That includes <slashCommands>Slash Commands</slashCommands>, <outgoingWebhooks>Outgoing Webhooks</outgoingWebhooks>, <interactiveMessages>Interative Messages</interactiveMessages> and <interactiveDialogs>Interative Dialogs</interactiveDialogs>.",
```

Comment:
there is a small typo in this line: `Interative` instead of `Interactive`

Reply:
2ae6ea7e77

Reply:
Good catch :+1: